packages/peregrine/src/hooks/__tests__/useDOMQuery.spec.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import {
+    TestValidComponent,
+    TestInvalidComponent
+} from '../__mocks__/useDOMQuery';
+
+describe('Unit testing useDOMQuery hook.', () => {
```

Comment:
> We try to avoid using describe

This is news to me, lol. For some files (like our `asyncActions`) that have a ton of functions that they're testing (and a handful of tests for each function), I've been wrapping each function's tests in a `describe` block. It also helps for having variables that are scoped only to that `describe` function.

Reply:
But for test files that only have a handful of test cases like this, I agree that the `describe` block is overkill.

Reply:
See, I told you we didn't make those conventions clear enough.

The reason we don't like to use them is that they're harder to find in the codebase using global text search.

```js
describe('the wheels on the bus', () => {
  it('go round and round', () => {
  })
})
```

The tree of describe/describe/it turns into a concatenated test description in the reporter. So this test appears in the log as `the wheels on the bus go round and round`, but if I search for that exact string I come up empty.

And the only thing we seem to get in return is indentation! So that's why. I'm open to counterargument as always.